content/01.main_content.md

```diff
@@ -1,35 +1,38 @@
-Recognition can greatly affect the trajectory of a scientist’s career since each form of recognition paves the way to future accolades.
-  Regardless of an individual’s merit, biases can skew which scientists are recognized.
-  Audits can combat this by identifying overlooked groups and holding the recognizing body accountable.
+Recognition by peers and the general public can greatly affect the trajectory of a scientist’s career.
+Any recognition--inclusion of one’s name in print as an expert, an invited lecture, or an award--paves the way to future accolades.
+Regardless of an individual’s merit, biases can skew which scientists are recognized.
   This is exemplified in a recent study of disparities in scientific journalism, which found that predicted East Asian names were under-quoted and under-mentioned [@doi:10.1101/2021.06.21.449261].
+  Regular monitoring of recognition practices can combat this by identifying overlooked groups and creating opportunities to hold the recognizing body accountable.
 
-Auditing efforts can reduce disparities.
-  Journalists found that self-audits of gender-based representation in their articles can help them approach parity [@https://www.theatlantic.com/technology/archive/2016/02/gender-diversity-journalism/463023/; @https://www.theatlantic.com/science/archive/2018/02/i-spent-two-years-trying-to-fix-the-gender-imbalance-in-my-stories/552404/].
+Auditing efforts have been shown to help reduce disparities.
+  For instance, several science journalists found that self-audits of gender-based representation in their articles can help them approach parity [@https://www.theatlantic.com/technology/archive/2016/02/gender-diversity-journalism/463023/; @https://www.theatlantic.com/science/archive/2018/02/i-spent-two-years-trying-to-fix-the-gender-imbalance-in-my-stories/552404/].
```

Comment:
This should be "several journalists" instead of "several **science** journalists" because the citations are from not just science journalists, Adrienne LaFrance also does politics.